Sources/ViewInspector/SwiftUI/ViewThatFits.swift

```diff
@@ -0,0 +1,24 @@
+import SwiftUI
+
+@available(iOS 16.0, macOS 13.0, tvOS 16.0, *)
+internal extension ViewType {
```

Comment:
Should this be public? For comparison: https://github.com/nalexn/ViewInspector/blob/0f872b50ceea6c26ac6904c29ac78899c3b86b6f/Sources/ViewInspector/SwiftUI/VStack.swift#L4-L9